core/src/test/scala/org/genivi/sota/core/jsonrpc/HttpTransportSpec.scala

```diff
@@ -36,11 +36,11 @@ class HttpTransportSpec extends JsonRpcSpecBase {
   }
 
   property("fails with error if status code indicates success but result is not a valid json") {
-    forAll(ResultStatusGen, Gen.alphaStr) { (statusCode: StatusCode, entity) =>
+    forAll(ResultStatusGen, Gen.alphaStr.suchThat(s => !s.isEmpty)) { (statusCode: StatusCode, entity) =>
```

Comment:
Well you could also just check if `entity` is empty then `res shouldBe a [NoContentExecpteion]`.